app/models/assignment.rb

```diff
@@ -26,7 +27,21 @@ class Assignment < ApplicationRecord
   scope :created_before, ->(max_time) { where('assignments.created_at < ?', max_time) }
   scope :created_after, ->(min_time) { where('assignments.created_at > ?', min_time) }
 
-  scope :ended, (-> { where('assignments.period_end < ?', Time.zone.today) })
+  scope :active, lambda {
+    started.where(
+      'assignments.period_end > ? OR assignments.period_end IS NULL',
+      Time.zone.today
+    )
+  }
+  scope :started, lambda {
+    where(
+      'assignments.period_start < ? AND assignments.period_start IS NOT NULL',
+      Time.zone.today
+    )
+  }
+  scope :ended, lambda {
```

Comment:
Also, out of curiosity, what is the advantage of using `lambda`?

Reply:
rubocop does not accept multiline lambdas like this:
```ruby
scope :example, (-> { where(something: 'xy') })  # is rubocophappy
scope :example, (-> {  # alltho this would AFAIR work, rubocop hates it
  where(x: 'd')
    ........
})
```

So thats why.
In general i dont like the syntax of those lambas anyway, because these three are all technicly lambda, but are syntactically inconsistent:
```ruby
scope :rara, (-> { xy })
scope :rara, (param)-> { where(x: param) }
scope :rara, lambda {
   ....
}
scope :rara, lambda { |param|
   ....
}
```

🤢